core/src/main/java/discord4j/core/object/entity/Member.java

```diff
@@ -314,6 +314,45 @@ public String getNicknameMention() {
                 .flatMap(ignored -> Mono.just(PermissionSet.all()))
                 .switchIfEmpty(Mono.zip(getEveryonePerms, getRolePerms, PermissionUtil::computeBasePermissions));
     }
+    
+    /**
+	 * Requests to determine if this member is higher in the role hierarchy than member.
+	 * This is determined by the positions of each of the members' highest roles.
+	 *
+	 * @param member The member who must be lower in the role hierarchy.
+	 * @return A {@link Mono} where, upon successful completion, emits {@code true} if this member is higher in the 
+	 * role hierarchy than member, {@code false} otherwise. If an error is received, it is emitted 
+	 * through the {@code Mono}.
+	 */
+    public Mono<Boolean> isHigher(Member member) {
+    	Mono<Integer> getHighestPosition = getRoles().flatMap(Role::getPosition).defaultIfEmpty(0).last();
```

Comment:
`defaultIfEmpty` I think is unnecessary since I think `getRoles` will always at the least return the everyone role. I may be wrong however. If that is not the case, ignore this.

Reply:
You already know but I will write this here: `getRoles` does not return the everyone role